schedule/yast/maintenance/offline_migration_15sp4.yaml

```diff
@@ -0,0 +1,29 @@
+name: schedule/yast/maintenance/offline_migration_15sp4
+description: |
+  This is for offline migraiton tests with unreleased maintenance updates
+  ORIGIN_SYSTEM_VERSION: '%HDDVERSION%'
```

Comment:
I don't think this will be replaced by the real value.

Reply:
removed.